src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/TdsParserStateObject.cs

```diff
@@ -558,8 +558,8 @@ internal bool TryInitialize(TdsParserStateObject stateObj, int columnsCount)
                 {
                     return false;
                 }
-                SqlClientEventSource.Log.TraceEvent("<sc.TdsParserStateObject.NullBitmap.Initialize|INFO|ADV> {0}#, NBCROW bitmap received, column count = {1}", stateObj.ObjectID, columnsCount);
```

Comment:
From the initial code, I assume that TraceEvent is different from AdvanceTrace. Is there any reason we the new version of AdvancedTraceEvent here?

Reply:
Looking at "ADV" in log, I would expect this is advance trace log.